make/test.mk

```diff
@@ -157,19 +157,30 @@ get-and-publish-host-operator:
 ###########################################################
 
 .PHONY: deploy-members
-deploy-members: create-member1 create-member2 get-and-publish-member-operator
+deploy-members: create-member1 create-member2 get-and-publish-member-operator create-member-resources
 
 .PHONY: create-member1
 create-member1:
 	@echo "Deploying member operator to $(MEMBER_NS)..."
 	$(MAKE) create-project PROJECT_NAME=${MEMBER_NS}
+	-oc label ns ${MEMBER_NS} app=member-operator
```

Comment:
We don't, I just find it useful for dev environments when the namespace isn't the default "toolchain-member-operator" one. I had a cleanup script that stopped working after the recent changes and it was because some of the labels were gone so I thought I'd add them. If they're problematic though I can remove it.

Reply:
they are not problematic at all - I was just curious. Would be worth adding a comment there so we know why they are there